### DISCUSSION TITLE
Q&A!!

### DISCUSSION THREAD

**Discussion by HACK3RY2J** [HACK3RY2J/Anon-SMS#12] [Q&A]
You can Start Asking Kind of question related to the tool..

 
> Be Friendly with Everyone.
> Its our Own Community. 
> Help Each other so other can help You.

**Comment by jovcho24** [comment]
![Screenshot_20210302_023638.jpg](https://user-images.githubusercontent.com/79677988/109544703-d6461400-7b02-11eb-8b94-4c78d3f164b3.jpg)

**Reply by jovcho24** [reply]
How to fix this sir?

**Reply by Bunny-on-github** [reply]
Just Re-install the script there are some requirements that are missing reinstalling will fix it
